### ISSUE TITLE
No Validation Set

### ISSUE THREAD

**Issue by 07Agarg** [LeeDoYup/FixMatch-pytorch#17]
Hi, For supervised and semi-supervised methods, it is generally advised to use a separate validation set. From the code, it looks like the best test set accuracy is reported.

Is there any specific reason that a separate validation set is not required.

**Comment by LeeDoYup** [comment]
In semi-supervised learning, constructing validation set is an open question, because the large size of validation set is not realistic. Thus, we follows the setting of the original FixMatch paper, and report the best performance.
